CMakeLists.txt

```diff
@@ -74,7 +76,7 @@ else()
     if(NOT LAR_CONTENT_LIBRARY_NAME STREQUAL "LArPandoraContent")
         set(LAR_CONTENT_LIBRARY_NAME "LArContent")
     endif()
-    project(${LAR_CONTENT_LIBRARY_NAME})
+    project(${LAR_CONTENT_LIBRARY_NAME} VERSION 3.22.12 LANGUAGES CXX )
```

Comment:
Hi @marcmengel , this part of the CMakeLists file is not active in the context of a LArSoft build. In Pandora's standalone build the version is set at line 90 in this version of the file, so  this update will need to be removed. Thanks!

Reply:
@AndyChappell, I would recommend not acting on Marc's PR just yet.  He is making changes that will be necessary for compatibility with Spack.  Once LArSoft fully migrates to *art* 3.09 (we're working on it), we'll have a better idea if other changes need to be made to the PR.